README.md

```diff
@@ -37,4 +37,18 @@ To the poor soul who takes my place; I beg of you, keep this up to date. It will
 
 Entity Relationship Diagram:<br>
 I want to move this off of Lucid Chart but for now there it shall lie.<br>
-[Entity Relationship Diagram](https://lucid.app/lucidchart/ff611a92-9484-4606-a11d-d142f54ee428/edit?viewport_loc=-2095%2C-830%2C4919%2C2456%2CMMHz0DLYv6yU&invitationId=inv_5c98f0bf-e268-4e23-afe7-0b5b771f8459)
+[Entity Relationship Diagram](https://lucid.app/lucidchart/ff611a92-9484-4606-a11d-d142f54ee428/edit?viewport_loc=-2095%2C-830%2C4919%2C2456%2CMMHz0DLYv6yU&invitationId=inv_5c98f0bf-e268-4e23-afe7-0b5b771f8459)<br>
+
+[Reduction to Tables](https://docs.google.com/document/d/1Dq0fAyz-GOTDRLL4TS9-DYxf0agS83yGcmjXoaB-jjU/edit?usp=sharing)<br>
+
+[Software Design Document](https://docs.google.com/document/d/1QV6rSz8Uj5yaqdDihSC-XkMlvNX5aUmDrTravnWL-ts/edit?usp=sharing)<br>
```

Comment:
Leave this as is for now, but at some point, all three of these should be migrated to this repository as markdown or draw.io files

Reply:
Sounds good